mgoutil/mgoutil.go

```diff
@@ -0,0 +1,39 @@
+// Package mgoutil provides various methods for working with the mgo.v2 package,
+// implementing various oft-used behaviors
+package mgoutil
+
+import (
+	"github.com/levenlabs/go-llog"
+	"gopkg.in/mgo.v2"
+)
+
+// Index is used in MustEnsureIndexes to convey all the information needed to
+// ensure a background index
+type Index struct {
+	Name string // human readable name for the index
```

Comment:
It's so the log entry doesn't just have a single letter to identify what the index is, just makes things easier to read. I'll embellish the comment a bit.